terraform/bin/config.sh

```diff
@@ -9,7 +9,6 @@ fi
 
 cd "$(dirname $BASH_SOURCE)/../"
 
-export APP_VERSION="${APP_VERSION:=$(grep -m 1 version ../pyproject.toml | awk -F ' = ' '{print $2}' | sed 's/"//g')}"
```

Comment:
The idea with this line is that it only sets the APP_VERSION if it's not already defined. If it's not defined then it uses the version in the pyproject.toml, otherwise if defined it uses that